### ISSUE TITLE
Can't use webtorrent.min.js directly from a webpage

### ISSUE THREAD

**Issue by mrozbarry** [feross/webtorrent#778]
* WebTorrent version: 0.91.0
* Node.js version: N/A
* Browser name/version (if using WebTorrent in the browser): Chrome 49.0.2623.112 (64-bit) on OSX Yosemite

Here is my html file:

```html
<!DOCTYPE html>
<html>
  <body>
    <script src="https://cdn.jsdelivr.net/webtorrent/latest/webtorrent.min.js"></script>
  </body>
</html>
```

More here:

![screen shot 2016-04-27 at 8 14 53 pm](https://cloud.githubusercontent.com/assets/566486/14871906/dd88258a-0cb4-11e6-8a00-b5060fc548dd.png)

I also tried using a version of `webtorrent.min.js` downloaded from npm (`node_modules/webtorrent/webtorrent.min.js`), with the same results.

I came across this while trying to get a minimal working case of webtorrent to narrow down where my issue was with #768.

**Comment by rickbeacham** [comment]
Try chrome canary?

**Comment by Lunik** [comment]
See #771 :)

**Comment by feross** [comment]
Fix published as 0.91.2
